server/src/pages/shot/view.js

```diff
@@ -436,6 +444,22 @@ class Body extends React.Component {
     </div>;
   }
 
+  onClickEdit() {
```

Comment:
The highlighter and pen both call the edit function as soon as they are clicked. It's just a general set up for both the tools, and not a tool in itself. Does it still need an event? @ianb